tests/functions/test_also.py

```diff
@@ -0,0 +1,63 @@
+from unittest.mock import Mock
```

Comment:
This is a new one for me, @sauln! Funnily enough, I've never mocked up anything in a test before. 

(Not a review, btw, just a comment.)